standard/flags_master_data.lua

```diff
@@ -1341,6 +1341,11 @@ local data = {
 		localised = 'African',
 		name = 'Africa',
 	},
+	['americas'] = {
```

Comment:
@[hjpalpha](https://github.com/hjpalpha) hjpalpha [on Oct 26, 2023](https://github.com/Liquipedia/Lua-Modules/pull/3432#discussion_r1372503021)
doesn't that break current usage?

[iMarbot](https://github.com/iMarbot) [on Oct 26, 2023](https://github.com/Liquipedia/Lua-Modules/pull/3432#discussion_r1372507437)
Potentially, however, pre-dating the Flags module, Template:Flag/americas was its own thing (but it happened to use NA flag file since there apparently wasn't something better).

[hjpalpha](https://github.com/hjpalpha) [on Oct 26, 2023](https://github.com/Liquipedia/Lua-Modules/pull/3432#discussion_r1372619814)
yeah but i think there are places relying on americas == usca flag

Reply:
checked on it and couldn't find cases where it causes issues